frontend/src/fetchers/collaboratorReports.js

```diff
@@ -0,0 +1,14 @@
+import join from 'url-join';
+import { get } from './index';
+
+export const getReports = async () => {
+  const reports = await get('/api/collaboration-report');
+  const json = await reports.json();
+  return json;
+};
+
+export const getCollaborators = async (region) => {
+  const url = join('/', 'api', 'users', 'collaborators', `?region=${region}`);
+  const collaborators = await get(url);
+  return collaborators.json();
+};
```

Comment:
this looks to be identical to the one in frontend/src/fetchers/activityReports.js, would it be better to reuse rather than copy/paste?